addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -126,4 +126,28 @@
     },
 ]);
 
+    // This fix addresses an issue that occurred in a website form containing
+    // the 'hide-change-model' attribute. Specifically, when a model-required
+    // field is selected, the alert message should not display an undefined
+    // action name.
+    wTourUtils.registerWebsitePreviewTour('model_required_field_should_have_action_name', {
```

Comment:
I cannot find the python code that launches this tour.
Before launching it, you might also want to create a specific job record for your test instead of relying on demo data.

Reply:
@bso-odoo A specific job record has already been created, so I initiated my tour from there.